File: PrimeiroProjeto/app/src/main/java/primeiroprojeto/linoveloso/com/MainActivity.java
package primeiroprojeto.linoveloso.com;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import java.util.Random;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void sortearNumero(View v){
        TextView textoResultado = (TextView) findViewById(R.id.textoResultado);
        int numero = new Random().nextInt(10);
        textoResultado.setText("O numero do resultado é "+ numero);
    }
}

File: ToggleeSwitch/app/src/main/java/com/cursoandroid/toggleeswitch/MainActivity.kt
package com.cursoandroid.toggleeswitch

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Switch
import android.widget.TextView
import android.widget.ToggleButton
import kotlinx.android.synthetic.main.activity_main.*
import org.w3c.dom.Text

class MainActivity : AppCompatActivity() {

    private val toggleSenha by lazy {
        findViewById<ToggleButton>(R.id.idToggleButton)
    }
    lateinit var switchSenha: Switch
    lateinit var textResultado: TextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        switchSenha = findViewById(R.id.idSwitch)
        textResultado = findViewById(R.id.idTextResult)
    }

    fun enviar(view: View?) {

        var resultado = textResultado.text
        if (switchSenha.isChecked) {
            resultado = "SWITCH ATIVADO"
        } else {
            resultado = "switch desativado"
        }
        if (toggleSenha.isChecked) {
            resultado += "\n Toggle ATIVADO"
        } else {
            resultado += "\n Toggle desativado"

        }

        textResultado.text = resultado

    }


}
